spec/classes/nginx_spec.rb

```diff
@@ -83,7 +84,7 @@
 
             it do
               is_expected.to contain_yumrepo('nginx-release').with(
-                'baseurl' => "https://nginx.org/packages/mainline/#{facts[:operatingsystem] == 'CentOS' ? 'centos' : 'rhel'}/#{facts[:operatingsystemmajrelease]}/$basearch/"
+                'baseurl' => "https://nginx.org/packages/mainline/#{%w[CentOS VirtuozzoLinux].include?(facts[:operatingsystem]) ? 'centos' : 'rhel'}/#{facts[:operatingsystemmajrelease]}/$basearch/"
```

Comment:
Same as above -- should these perhaps have included `Oracle` as well? Might it be better to simply reverse the test, is only RedHat proper uses `rhel` and the rest should use `centos`?

```ruby
'baseurl' => "https://nginx.org/packages/mainline/#{facts[:operatingsystem] == 'RedHat' ? 'rhel' : 'centos'}/#{facts[:operatingsystemmajrelease]}/$basearch/"
```

Reply:
See https://github.com/voxpupuli/puppet-nginx/pull/1354/files#r347156132